examples/rllib_training.ipynb

```diff
@@ -232,7 +233,7 @@
     "        metrics_episode_collection_timeout_s=180,\n",
     "    )\n",
     "    .checkpointing(export_native_model_files=True)\n",
-    "    .framework(framework=\"tf2\")\n",
+    "    .framework(framework=\"torch\")\n",
```

Comment:
Let's leave this as `tf2` to keep the commit cleaner. I have been using the `tf2` backend because in older versions of Ray I was unable to successfully load the torch model files that it saved, but I'd be curious if that has been fixed since then.